documentation/book/ref-topic-operator-usage-recommendation.adoc

```diff
@@ -5,11 +5,38 @@
 [id='topic-operator-usage-recommendations-{context}']
 = Topic Operator usage recommendations
 
-* Be consistent and always operate on `KafkaTopic` resources or always operate on topics directly. Avoid routinely using both methods for a given topic.
-* When creating a `KafkaTopic` resource:
-** Remember that the name cannot be changed later.
-** Choose a name for the `KafkaTopic` resource that reflects the name of the topic it describes.
-** Ideally the `KafkaTopic.metadata.name` should be the same as its `spec.topicName`. To do this, the topic name will have to be a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^].
-* When creating a topic:
-** Remember that the name cannot be changed later.
-** It is best to use a name that is a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^], otherwise the operator will have to modify the name when creating the corresponding `KafkaTopic`.
+When working with topics, be consistent and always operate on either `KafkaTopic` resources or topics directly. Avoid routinely switching between both methods for a given topic.
+
+Use topic names that reflect the nature of the topic, and remember that names cannot be changed later.
+
+If creating a name directly for a topic, use a name that is a valid {KubernetesName} resource name, otherwise the Topic Operator will need to modify the name when creating the corresponding `KafkaTopic`.
+
+.Kafka topic naming conventions
+
+Kafka and {KubernetesName} impose their own validation rules for the naming of `KafkaTopic` and `KafkaTopic.metadata.name` respectively.
+So there are valid naming conventions for each which are invalid in the other.
+
+The `spec.topicName` property provides a way to create a name for a topic in Kafka that would be a valid name for `KafkaTopic` in {KubernetesName}, link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[(according to the recommendations for identifiers and names in {KubernetesName})^].
+
+The `spec.topicName` property inherits Kafka naming validation rules:
+
+* The name must not be longer than 249 characters
+* Valid characters for Kafka topics are ASCII alphanumerics, ‘.’, ‘_’, and ‘-‘
```

Comment:
Strictly speaking it can't be `.` or `..` either. I suppose we should mention that.

Reply:
* The name cannot be `.` or `..`, though `.` can be used in a name, such as `exampleTopic.` or `.exampleTopic`